dev/ci/ci-hott.sh

```diff
@@ -7,4 +7,4 @@ HoTT_CI_DIR=${CI_BUILD_DIR}/HoTT
 
 git_checkout ${HoTT_CI_BRANCH} ${HoTT_CI_GITURL} ${HoTT_CI_DIR}
 
-( cd ${HoTT_CI_DIR} && ./autogen.sh && ./configure && make -j ${NJOBS} )
+( cd ${HoTT_CI_DIR} && ./autogen.sh && ./configure && ./etc/coq-scripts/timing/make-pretty-timed-or-error.sh -j ${NJOBS} && make validate )
```

Comment:
I think I'd rather have the timing script merged in Coq and supported there; my worry is state-space explosion; even if it is unlikely, having an outside script running in the core Travis part introduces yet another factor to check when a problem arises.